dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/DataverseUserPage.java

```diff
@@ -569,6 +569,10 @@ public AuthenticationProvider getUserAuthProvider() {
         return userAuthProvider;
     }
 
+    public String getUserNotificationsLanguageTitle() {
+        return settingsWrapper.getConfiguredLocaleName(currentUser.getNotificationsLanguage());
```

Comment:
`getConfiguredLocaleName` will return name in native language (doesn't take user locale into account) that is `English` and `Polski`. Don't know if it is something that we want. If we would Have Locale object then we could get language name in user specified language (see: `Locale.getDisplayName(Locale)` and `DataverseSession.getLocale`).
Even if we wouldn't operate on Locale then we would have to have some fallback. We can't assume that `getConfiguredLocaleName` will exist for `currentUser.getNotificationsLanguage`

Reply:
I would kinda expect this to be in native language - the way you have it for example on wikipedia.org or what we have in dataverse when changing app language.

Reply:
Discussed with @Wojtek - language is now translated in accordance with app locale.